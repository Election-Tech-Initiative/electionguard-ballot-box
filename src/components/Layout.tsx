import React from 'react'
import { Redirect, Route, Switch } from 'react-router-dom'

import StartPage from '../pages/StartPage'
import NotFoundPage from '../pages/NotFoundPage'

const Layout = () => {
  return (
    <Switch>
      <Redirect exact path="/" to="/start" />
      <Redirect exact path="/index.html" to="/start" />
      <Route path="/start" exact component={StartPage} />
      <Route path="/:path" component={NotFoundPage} />
    </Switch>
  )
}

export default Layout
