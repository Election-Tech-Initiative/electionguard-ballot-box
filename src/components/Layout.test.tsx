import React from 'react'

import { render } from '../../test/testUtils'

import Layout from './Layout'

it('can renders Layout', () => {
  const { container } = render(<Layout />, {})
  expect(container.firstChild).toMatchSnapshot()
})
