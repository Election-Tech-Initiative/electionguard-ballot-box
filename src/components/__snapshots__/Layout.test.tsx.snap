// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can renders Layout 1`] = `
<div
  class="sc-fjdhpX eFcaR"
>
  <div
    class="sc-bZQynM bsmBBS"
  >
    <h1>
      Ballot Box
    </h1>
    <hr
      class="sc-EHOje YINzU"
    />
    <p>
      Select Ballot Box Type
    </p>
    <div
      class="sc-gzVnrw cOUDlP"
    >
      <div
        class="sc-bxivhb woZhC"
      >
        <button
          aria-label="Select cast as ballot box type"
          class="sc-bwzfXH eHmtkf"
          type="button"
        >
          Cast
        </button>
      </div>
      <div
        class="sc-bxivhb woZhC"
      >
        <button
          aria-label="Select spoil as ballot box type"
          class="sc-bwzfXH eHmtkf"
          type="button"
        >
          Spoil
        </button>
      </div>
    </div>
  </div>
  <img
    alt="Election Guard Logo"
    class="sc-jzJRlG bhYxid"
    src="/images/rock-county-wisconsin-logo.png"
  />
</div>
`;
